### DISCUSSION TITLE
Can't add packs to lunii

### DISCUSSION THREAD

**Discussion by gael4910** [marian-m12l/studio#187] [Q&A]
Hello,
I got a brand new lunii (v2) to replace a broken V1. I tried to put my own packs with studio 0.3.0 beta but i can't slide packs to the left menu. I see my official packs on the left, my own pack on the right but I get an icon that looks like "no parking sign" when trying to upload it on my box. I tried to reload studio.bat several times and reboot PC.

**Comment by gael4910** [comment]
I finally got it, studio only works with home made packs.

**Reply by vpoulailleau** [reply]
I also use STUdio with official packs. You're even able to edit them if you want. But STUdio was created to make your own packs.

You can manage both homemade and official story packs on your lunii device with STUdio.
